components/cluster-secret-store/staging/cluster-secret-store-patch.yaml

```diff
@@ -0,0 +1,26 @@
+# cluster-secret-store-patch.yaml
```

Comment:
I don't understand this change. Why is this file needed? Why is it added now (I would have expected an update that removes a reference to tekton-chains instead)? Why is it referencing things that are outside our control?

It feels to me that the changes in this dir should have been a separate PR.

Reply:
I'll try to clarify why a new file is added. So, first we want to rollout the changes around removal of `tekton-chains` ns in the staging. 
i. Now, this [ClusterSecretStore `appsre-stonesoup-vault`](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/cluster-secret-store/base/appsre-stonesoup-vault-secret-store.yaml) is in the base of the kustomization and therefore it allows the `tekton-chains` ns both in staging and in cluster to use the vault.
ii. If I just remove [this line](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/cluster-secret-store/base/appsre-stonesoup-vault-secret-store.yaml#L36), although that would make sure that `tekton-chains` ns which will be gone in future won't be allowed to use vault but at the same time it's going to affect both prod & stag.
iii. And therefore in accordance with [Gabe's ask](https://github.com/redhat-appstudio/infra-deployments/pull/2364#discussion_r1317257338) I came up with this patch. I don't know if there's an easier way to patch the resource to just exclude a specific ns.

This change will be temporary though. As soon as we verify this in staging, I'll raise a PR to update base file to remove `tekton-chains` ns.